openapi.json

```diff
@@ -14,6 +14,60 @@
 	}],
 	"paths": {
 		"/load-test": {
+			"get": {
+				"tags": ["load-tests"],
+				"summary": "List all the load tests",
+				"operationId": "listLoadTest",
+				"parameters": [
+					{
+						"name": "phase",
+						"in": "query",
+						"description": "Filter the result by phase. Possible values: creating, starting, running, finished, errored",
+						"schema": {
+							"type": "string",
+							"enum": [
+								"creating",
+								"starting",
+								"running",
+								"finished",
+								"errored"
+							]
+						},
+						"example": "running"
+					},
+					{
+						"name": "tags",
+						"in": "query",
+						"description": "Filter the result by tags, value is in format: tag1:value1,tag2:value2",
+						"schema": {
+							"type": "string"
+						},
+						"example": "department:platform,team:kangal"
+					}
```

Comment:
It's not missing the `limit` field here?

Reply:
sharp eyes 👍